jenkins-pipelines/rolling-upgrade-ami-arm.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+rollingUpgradePipeline(
+    backend: 'aws',
+    base_versions: '',  // auto mode
+    linux_distro: 'centos',
```

Comment:
I think AMI's are using ubuntu as a base. So maybe this is not used at all in ami tests - thus could be removed.

Reply:
that is a good point... need to check last master jobs, how or if they are passing, because for some time already we have Ubuntu as the base OS for all Scylla images

Reply:
you removed it from the original AMI job, so please, remove it here as well

Reply:
Fixed (removed)